api/src/pcapi/core/mails/transactional/educational/eac_new_request_made_by_redactor_to_ac.py

```diff
@@ -0,0 +1,32 @@
+from pcapi.core import mails
+import pcapi.core.educational.models as educational_models
+from pcapi.core.mails import models
+from pcapi.core.mails.transactional.sendinblue_template_ids import TransactionalEmail
+
+
+def send_new_request_made_by_redactor_to_ac(request: educational_models.CollectiveOfferRequest) -> bool:
```

Comment:
"AC" est l'acronyme en français, on utilise plutôt "pro" dans notre code.

```suggestion
def send_new_request_made_by_redactor_to_pro(request: educational_models.CollectiveOfferRequest) -> bool:
```

idem pour `get_data_request_made_by_redactor_to_ac()` plus bas.

Reply:
@borisLeBonPro ? Tu as mergé cette PR sans prendre en compte ce commentaire.